frontend-wasm2/src/translation_utils.rs

```diff
@@ -147,3 +150,21 @@ pub fn sig_from_func_type(
         visibility,
     }
 }
+
+/// Parse `builtin::ComponentId` from the Wasm CM component name in the
+/// `namespace:package/interface@version` format
+pub fn parse_component_id(wasm_component_name: String) -> builtin::ComponentId {
```

Comment:
This function can probably just be replaced with `wasm_component_name.parse::<builtin::ComponentId>().unwrap()`, since `ComponentId` implements `FromStr`.

Reply:
Thanks! I removed the function.